doc/wg/core/notes/core-notes-2023-11-10.md

```diff
@@ -0,0 +1,89 @@
+Tock Core Notes 2023-11-10
+==========================
+
+## Attendees
+ - Branden Ghena
+ - Leon Schuermann
+ - Alyssa Haroldsen
+ - Hudson Ayers
+ - Brad Campbell
+ - Johnathan Van Why
+ - Alexandru Radovici
+ - Amit Levy
+
+## Updates
+
+### Newlib
+ * Brad: For libtock-c newlib packaging process. Last time we talked about this, there were some questions about where to store the artifacts and how reproducible it is. I now have a docker file that is pretty close to being able to build the artifact both for us and in our CI. It seems that newlib is not as extensively tested as I had hoped it was. 
+ * Hudson: The reason you need a docker container is that Ubuntu 22 shipped with GCC 10?
+ * Brad: We just wanted a way that even if the artifacts disappear, someone could go back and rebuild newlib for some moment in time. As a side effect, and why I really wanted it, is that it lets me build it at all. There is indeed a conflict with newer and older versions of GCC. For RISC-V newer versions of GCC build a newlib that doesn't work with old versions.
+
+ * Leon: I have a CW310 board, which is a reference for the OpenTitan RISC-V FPGA. Zerorisc provided it. I may or may not be able to provide remote access to that board for testing changes.
+
+### Syscall Tunnel
+ * Johnathan: I'm building a proof-of-concept for what I call the "syscall tunnel" for libtock-rs. It forwards system calls over a UART to allow apps to run on a Host computer and forward requests to Tock on a microcontroller.
+ * Johnathan: I'm running into a rough bug here, which might be undefined behavior? I can't reproduce in Miri.
+ * Branden: What's the purpose behind that?
+ * Johnathan: Testing infrastructure. Make calls to a tock board to perform operations. Then run a check on some other board to see that it received it.
+ * Brad: So the Tock board runs some userland app that receives the syscalls? (yes)
+ * Johnathan: Right now it doesn't work with the "pin-based allow API"
```

Comment:
I probably said "doesn't work without", but "Right now it needs the pin-based allow API" is probably a less confusing way to word the notes.

Reply:
Good catch! I didn't (still don't) know that that is, so it was easier for me to mess up your meaning there.